src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -148,7 +148,6 @@ public static IHostBuilder AddScriptHostCore(this IHostBuilder builder, ScriptAp
                 services.AddTransient<IExtensionsManager, ExtensionsManager>();
                 services.TryAddSingleton<IHttpRoutesManager, DefaultHttpRouteManager>();
                 services.TryAddSingleton<IMetricsLogger, MetricsLogger>();
-                services.TryAddSingleton<IScriptJobHostEnvironment, ConsoleScriptJobHostEnvironment>();
```

Comment:
>services.TryAddSingleton<IScriptJobHostEnvironment, ConsoleScriptJobHostEnvironment>(); [](start = 16, length = 87)

Should this be replaced with something like
services.TryAddSingleton<IScriptHostManager, ConsoleScriptHostService>(); 

where ConsoleScriptHostService's RestartHostAsync would just mean _applicationLifetime.StopApplication() for the purpose of a console app ?

Reply:
I don't think we need to add anything here for now